plugins/lua/sort.lua

```diff
@@ -211,7 +506,7 @@ SquadAssignmentOverlay.ATTRS{
     default_pos={x=-33, y=40},
     default_enabled=true,
     viewscreens='dwarfmode/UnitSelector/SQUAD_FILL_POSITION',
-    frame={w=63, h=7},
+    frame={w=65, h=9},
```

Comment:
This is too tall for a minimum size window, but no need to change it. I need to rework the layout of this overlay anyway to add filter toggles (e.g. hide nobles, hide units already in squads, etc.)

Reply:
I am chaning it here to fit options.

Reply:
that's fine for now. I need to add more stuff too, though, and will likely have to rearrange into a vertical panel instead of the current horizontal one. this PR is fine as-is